Bio/PDB/ccealignmodule.c

```diff
@@ -461,9 +619,11 @@ findPath(
             }
         }
 
+        const double zScore = zScoreBuffer[o];
```

Comment:
Hm, with this getting a little more complex, I wonder if we shouldn't return a struct for each alignment with pathA, pathB, zscore, etc.